api-reference/introduction.mdx

```diff
@@ -4,7 +4,7 @@ description: 'Technical documentation for Gladia API'
 ---
 
 <Note>
-  If you're not looking for technical API reference documentation, you can check our [Quickstart guide](/chapters/get-started/pages/transcribe-audio).
+  If you're not looking for technical API reference documentation, you can check our [Quickstart guide](/chapters/introduction/getting-started).
```

Comment:
Could have a CI step to automatically detect if an URL path is no longer valid?

Reply:
`mintlify broken-links`

Reply:
Confusing, but it's being ran under `npm test` ci step

![image](https://github.com/user-attachments/assets/92d6cbb8-bc21-4bf5-afa3-f9eb8d8e2021)